scripts/setup-benchmarks/install_ycsb.sh

```diff
@@ -0,0 +1,13 @@
+# Update and install required packages
+apt update
+
+# Download YCSB
+curl -O --location https://github.com/brianfrankcooper/YCSB/releases/download/0.17.0/ycsb-0.17.0.tar.gz
```

Comment:
please install `ycsb` to the same location that the other benchmarks do like in https://github.com/utcs-scea/KernMLOps/blob/0614a75224479808e21321eac337c3ad9151f578/scripts/setup-benchmarks/setup-gap.sh#L11